src/provisioner/const.py

```diff
@@ -13,12 +13,14 @@
 # For any questions about this software or licensing,
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 
+from pathlib import Path
+
 SERVICE_NAME = "cortx_setup"
 APP_NAME = "provisioner"
 TMP_LOG_PATH = "/tmp/%s" % APP_NAME
 DEFAULT_LOG_PATH = "/var/log/cortx/%s" % APP_NAME
 CONFIG_PATH="/etc/cortx/config"
-MACHINE_ID_PATH = "/etc/cortx/config/machine-id"
+MACHINE_ID_PATH = Path("/etc/cortx/config/machine-id")
```

Comment:
this Path conversion should  have been at the caller side

Reply:
Instead of calling Path() by caller side everytime, if its actually a path and getting used everywhere, it should belong in const.py right?

Reply:
yes, if everywhere it is being treated as a path directly and nothing else then it is fine